lib/reporting/command_line_options.rb

```diff
@@ -1,5 +1,8 @@
+require 'reporting/cloudwatch_query_time_slice'
+
 module Reporting
   class CommandLineOptions
+    include Reporting::CloudwatchQueryTimeSlice
```

Comment:
since we're calling `parse_duration` as a static method, this `include` does not really do anything? I'd remove it
```suggestion
```